content/blog/2018/02/2018-02-27-browser-usage.adoc

```diff
@@ -0,0 +1,194 @@
+
+
+=== Fun facts:
+
+* Netscape 4.8 was still used by 15 visitors over the years.
+* 6 visitors used their Nintendo DS.
+|
+|===
+
+
+[cols="a,a"]
+|=== 
+|
+== StatCounter
+Let's now compare our previous results with the same time period from http://gs.statcounter.com/about[statcounter] who argues to collect data from 2million websites.
+Statcounter allow us to display browsers stats based on 4 different categories, 'All platform', 'Desktop', 'Tablet' or 'Mobile'. +
+For this comparison, I am interested by 'All platform' and of course 'Desktop' as 92.28% of Jenkins visitors use a desktop.
+
+|
+|
+.All Platform Browser Market Share
+[cols="1,9,1",width="100%"]
+!===
+!#!Browser ! Use
+
+! 1. 
+! Chrome
+! 54.13%
+
+! 2. 
+! Safari
+! 14.50%
+
+! 3.
+! UC Browser
+! 8.25%
+
+! 4. 
+! Firefox
+! 6.15%
+
+! 5.
+! IE
+! 3.89%
+
+! 6.
+! Opera 
+! 3.85%
+
+! 7.
+! Edge
+! 1.85%
+
+! 8.
+! Others
+! 1.76%
+
+!===
+|
+=== All Platform Browser Market Share
+Excepted Chrome who still leads the ranking with "only" 54%, all the others actors are totally shuffled. +
+Safari moved to the 2nd place with 14.50% which is a growth of 225.15%. +
+'UC Browser' appeared in the top 3 with 8.25% where only 0.10% of Jenkins visitors were using UC browser. +
+Firefox moved to the 4th places with only 6.15%. +
+It's interesting how this browser market share doesn't reflect Jenkins visitors' habits.
+
+http://gs.statcounter.com/browser-market-share#monthly-201701-201801-bar[Source]
+
+|
+=== Desktop Browser Market Share
+Now at a first glance, results look more similar to the Jenkins visitors; excepted for Chrome and Firefox which shed some percentage in favor of Internet Explorer and Edge. +
+This promote IE to the 3rd place with 8.86% of usage, and downgrade Safari to the 4th place with 6,44%. +
+Compare to Jenkins results, Edge is still at the 5th position. +
+It's very interesting to see how this ranking is so different from the "All platform" one but follows the same pattern coming from Jenkins visitors. +
+This highlights how browser choices are strongly influenced by the platform used and contrary to a general http://gs.statcounter.com/platform-market-share/desktop-mobile-tablet[trend], Jenkins visitors remain desktop users.
+
+http://gs.statcounter.com/browser-market-share/desktop/worldwide#monthly-201701-201801-bar[Source]
+
+|
+.Desktop Browser Market Share
+[cols="1,3,1",width="30%"]
+!===
+!#!Browser ! Use
+
+! 1. 
+! Chrome
+! 63.59%
+
+! 2.
+! Firefox
+! 13.75%
+
+! 3. 
+! IE
+! 8.86%
+
+! 4.
+! Safari
+! 5.50%
+
+! 5. 
+! Edge
+! 4.35%
+
+!===
+|===
+== Conclusion
+These observations should be carefully interpreted as it may be overestimated or underestimated (depending on various https://en.wikipedia.org/wiki/Usage_share_of_web_browsers#Accuracy[factors])  and doesn't take into consideration local specificities.  +
+
+Jenkins visitors are a majority of desktop users, without any doubts. +
```

Comment:
I doubt ;) Let's make the site working on Mobile first. It could help us to grow traffic as well

IMHO this sentence should be amended a bit so that it does not implicitly say "we do not care about non-desktop visitors". It would make sense to say that we still DO care about mobile users and welcome contributions for this area.

Reply:
Good point